indexers/vector/FaissIndexer/tests/test_faissindexer.py

```diff
@@ -77,6 +77,32 @@ def test_faiss_indexer_known(metas):
         assert idx.shape == (4, 2)
         np.testing.assert_equal(indexer.query_by_id([7, 4]), vectors[[3, 0]])
 
+    # update
```

Comment:
Can you duplicate the test adding the update feature? to have it isolated?

Reply:
Done